polypomo

```diff
@@ -133,6 +138,57 @@ class Status:
             self.timer = Timer(self.worktime)
 
 
+@contextmanager
+def setup_conn():
+    global DATABASE_PATH
+    # check if the database exist
+    if (os.path.exists(DATABASE_PATH)):
+        print('database already exist')
+        conn = sqlite3.connect('polypomoDB.db')
+        cur = conn.cursor()
+        try:
+            yield cur
+        finally:
+            conn.commit()
+            conn.close()
+    else:
+        print('initiate database')
+        conn = sqlite3.connect('polypomoDB.db')
+        cur = conn.cursor()
+        cur.execute('''CREATE TABLE data
+               (days text, worked_time text)''')
```

Comment:
This will need to be restructured following my comments in the pull request.
We'll want to store start/stop times in the database and compute a total day-time through external tools.